transactions.sql

```diff
@@ -0,0 +1,39 @@
+-- update the animals table by setting the species column to unspecified.
+-- Then roll back the change.
+
+BEGIN;
+update animals set species = 'unspecified';
+ROLLBACK;
+
+
+-- Update the animals table by setting the species column to digimon for all animals 
+-- that have a name ending in mon.
+-- Update the animals table by setting the species column to pokemon for all animals 
+-- that don't have species already set.
+
+BEGIN;
+update animals set species = 'digimon' where name like '%mon';
+update animals set species = 'pokemon' where species is NULL;
```

Comment:
Please add a query to verify explicitly that change was made before and after the commit.